src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      action="http://localserver:3000/api"
```

Comment:
add correct action from repo to this task

Reply:
I don`t know which  action="' should be implemented as there was no mention in the task

Reply:
My code doesn`t have this
<h1>Search bar airbnb</h1>